### DISCUSSION TITLE
Implementing Custom Node View

### DISCUSSION THREAD

**Discussion by patrickcorrigan** [ueberdosis/tiptap#2055] [General Questions]
I'm implementing a Node View in Vue, 

It contains multiple input fields. When the node view is created I'm struggling to focus on the first field. What is the best way to go about this? Is there a way to define a focus point in a Node View

**Comment by philippkuehn** [comment]
No, the content of node views is like a black box fir tiptap. you have to create something by yourself. 

try to focus your input on the created hook of your vue component.

**Reply by patrickcorrigan** [reply]
I tried that but for some reason it didn't work :(

**Reply by philippkuehn** [reply]
hmm. can you try to create a reproducable codesandbox?
